.github/workflows/build.yml

```
@@ -226,13 +222,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the windows admin password
+        # pulled from AWS Parameter Store. Return the admin password for
+        # Packer to consume.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
         # This runs through the AMI creation process but does not
         # actually create an AMI
         run: |
+          PKR_VAR_admin_pass=\
+          '${{ steps.process-user-data.outputs.pass }}' \
           packer build -timestamp-ui \
-            -var skip_create_ami=true \
-            src/packer.pkr.hcl
+          -var skip_create_ami=true \
+          src/packer.pkr.hcl
```

It's best to not commit spacing changes like this, since it will always show up as an unnecessary difference from [the skeleton](https://github.com/cisagov/skeleton-packer/blob/d2ed09c808fc142db2c75d80b737f571be5fe83a/.github/workflows/build.yml#L233-L235).
```suggestion
            -var skip_create_ami=true \
            src/packer.pkr.hcl
```

---

resolved with [abc45f6](https://github.com/cisagov/windows-commando-packer/pull/10/commits/abc45f6ff193bb1e977d69f5712847e2a317111b)